src/schema_generators/generators.cljc

```diff
@@ -223,3 +224,17 @@
   "Sample a single element of low to moderate size."
   [& generator-args]
   (generators/generate (apply generator generator-args) 10))
+
+(s/defn generate-alt :- s/Any
+  "Sample a single element
+
+  Providing the same random seed will give the same result"
+  ([schema random-seed] (generate-alt schema random-seed 10))
+  ([schema random-seed size] (generate-alt schema random-seed size {} {}))
+  ([schema :- Schema
+    random-seed
+    size :- s/Int
+    leaf-generators :- LeafGenerators
+    wrappers :- GeneratorWrappers]
+   (let [gen (generator schema leaf-generators wrappers)]
+     (rose/root (generators/call-gen gen random-seed size)))))
```

Comment:
It is unfortunate to depend on internal `clojure.test` logic (like with `(rose/root (generators/call-gen ..))`), but it is pretty contained, so I think it is worth it

Reply:
It looks like `test.check` exposes these as arguments to `generators/generate`, no?

Reply:
In recent releases, yes.